### ISSUE TITLE
"Average AI" radio button lacks keyboard shortcut

### ISSUE THREAD

**Issue by amccarthy1** [JRJurman/domination-testing#46]
"Human", "Easy AI", and "Hard AI" all have hotkeys associated with them (Alt-U, Alt-E, and Alt-H, respectively). However, "Average AI" does not have such a hotkey. For accessibility and consistency purposes, this option should have a hotkey as well.
